src/Cli/OrgCommands.php

```diff
@@ -70,7 +70,12 @@ class OrgCommands extends \Robo\Tasks implements ConfigAwareInterface, LoggerAwa
      *
      * @return Consolidation\OutputFormatters\StructuredData\RowsOfFields
      */
-    public function orgAnalyze($org, $options = ['as' => 'default', 'format' => 'table'])
+    public function orgAnalyze($org, $options = [
+        'as' => 'default',
+        'format' => 'table',
+        'only-public' => false,
+        'no-forks' => false,
```

Comment:
Might be clearer to use `'forks' => true,`, which will also add a `--no-forks` option (to set the value to false)